examples/vanilla-ts-esm/public/mux-player-audio.html

```diff
@@ -48,6 +48,16 @@ <h1><a href="/">Elements</a></h1>
       playback-id="O4h5z00885HEucNNa1rV02wZapcGp01FXXoJd35AHmGX7g"
     ></mux-player>
 
+    <br>
+
+    <mux-player
+      audio
+      stream-type="on-demand"
+      playback-id="O4h5z00885HEucNNa1rV02wZapcGp01FXXoJd35AHmGX7g"
+      style="max-width: 480px"
+      poster="https://image.mux.com/23s11nz72DsoN657h4314PjKKjsF2JG33eBQQt6B95I/thumbnail.webp?width=640"
```

Comment:
suggestion(non-blocking): maybe have two examples here: one with poster and one without? (can be on the same page imo)